src/main/resources/treeformats.yml

```diff
@@ -51,6 +51,17 @@ formats:
     version:
       regex: "@([^@\\s]+)(\\s\\S+)?$"
       group: 1
+  - format: nuget
+    description: "Nuget tree output"
+    tool: "See https://github.com/Brend-Smits/NugetTree"
```

Comment:
My intention was to put context and explanation in the `description` field, and provide a sample invocation of the tree generation tool in the `tool` field. (This helps in the copy-paste engineering that I have come to like on the command line.)

Reply:
Sounds good.